models/topics.models.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query(`SELECT slug, description FROM topics`).then(({ rows }) => {
```

Comment:
This query works, but you could say `SELECT * FROM topics` which shows to any other developers that you're getting everything from the topics table 👍